nx.json

```diff
@@ -187,6 +187,9 @@
       ]
     },
     "build-source": {
```

Comment:
Maybe simpler to rename the `build-source` from the `showcase` to `compile` to be align with the other packages

Reply:
I'd say it might be better to keep it as close as possible as the `angular.json` generated from new apps